source/scripting_v3/GTA/World.cs

```diff
@@ -236,6 +236,20 @@ public static float GravityLevel
 
 		#endregion
 
+		#region Wind
+		public static float WindSpeed
+		{
+			get => Function.Call<float>(Hash.GET_WIND_SPEED);
+			set
+			{
+				if (value >= 0f && value <= 12.0f)
```

Comment:
I think wind speed should be changed in all cases but with the clamped value, just like `Ped.Sweat` does.
https://github.com/crosire/scripthookvdotnet/blob/4b6d05aea11eab20f8a7ab28fd30277b1f492a99/source/scripting_v3/GTA/Entities/Peds/Ped.cs#L194